backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -62,4 +63,38 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 	public Map<UUID, Shop> getAllShops() {
 		return shopMap;
 	}
+
+	public Token addTokenToShop(UUID shopId) {
+
+		if (shopMap.get(shopId).getStatus() == ShopStatus.ACTIVE) {
+
+			Integer lastToken = shopIdToLastAllotedTokenMap.get(shopId).incrementAndGet();
+			UUID uuid = UUID.randomUUID();
+
+			Token newToken = new Token();
+			newToken.setTokenId(uuid);
+			newToken.setShopId(shopId);
+			newToken.setTokenNumber(lastToken);
+			newToken.setStatus(Token.Status.ACTIVE);
+
+			tokenMap.put(uuid, newToken);
+
+			if (lastToken != 1) {
+
+				List<Token> newTokenList = shopIdToListOfTokensMap.get(shopId);
+				newTokenList.add(newToken);
+				shopIdToListOfTokensMap.put(shopId, newTokenList);
+			}
+
```

Comment:
Nit. 
"} else {"

instead of the extra lines.

Reply:
Done